src/backend/utils/systeminfo/heroicVersion.ts

```diff
@@ -6,9 +6,7 @@ function getHeroicVersion(): string {
   // FIXME: Currently ESLint doesn't know that these are type-checked. I believe
```

Comment:
I think this comment can be removed now? sound like it was a FIXME for that eslint-disable